uxarray/_exodus.py

```diff
@@ -221,7 +221,7 @@ def _encode_exodus(ds, ds_var_names, outfile=None):
     # process face nodes, this array holds num faces at corresponding location
     # eg num_el_all_blks = [0, 0, 6, 12] signifies 6 TRI and 12 SHELL elements
     num_el_all_blks = np.zeros(ds[ds_var_names["nMaxMesh2_face_nodes"]].size,
-                               "i4")
+                               "i8")
```

Comment:
Since face nodes are always natural numbers, would it make more sense to use unsigned integers `u8` instead of signed integers?

Reply:
Additionally, do we need 64 bit precision for our face nodes? Using a 32 bit unsigned integer would cap our max number of face nodes at slightly over 4 billion. Is there any scenario where we would have a grid with over 4 billion face nodes?